.github/pull_request_template.md

```diff
@@ -1,18 +1,15 @@
-## Important (read before submitting)
-We are currently preparing for the upcoming 1.0 GA release. Pull requests that are not aligned with
-the current roadmap and are not aimed at stabilizing and preparing the Collector for the release will
-not be accepted.
-
-_Delete the preceding paragraph before submitting._
-
-**Description:** <Describe what has changed.>
 <!--Ex. Fixing a bug - Describe the bug and how this fixes the issue.
 Ex. Adding a feature - Explain what this achieves.-->
+#### Description
 
-**Link to tracking Issue:** <Issue number if applicable>
+<!-- Issue number if applicable -->
+#### Link to tracking Issue
```

Comment:
```suggestion
#### Link to tracking issue
```
Nit: I don't know why issue has been capitalized in the past, but feel free to ignore if it's intended to be that way.